public/app/plugins/datasource/prometheus/language_utils.ts

```diff
@@ -111,3 +113,34 @@ export function expandRecordingRules(query: string, mapping: { [name: string]: s
   const rulesRegex = new RegExp(`(\\s|^)(${ruleNames.join('|')})(\\s|$|\\(|\\[|\\{)`, 'ig');
   return query.replace(rulesRegex, (match, pre, name, post) => `${pre}${mapping[name]}${post}`);
 }
+
+/**
+ * Adds metadata for synthetic metrics for which the API does not provide metadata.
+ * See https://github.com/grafana/grafana/issues/22337 for details.
+ *
+ * @param metadata HELP and TYPE metadata from /api/v1/metadata
+ */
+export function fixSummariesMetadata(metadata: PromMetricsMetadata): PromMetricsMetadata {
+  if (!metadata) {
+    return metadata;
+  }
+  const summaryMetadata: PromMetricsMetadata = {};
```

Comment:
nit: a very minor and optional nit is to move all the suffices and types to constants.